dataset_features_provider.py

```diff
@@ -848,6 +848,19 @@ def validate_fields(self, feature):
             input_value = feature['properties'][attr]
             value = None
 
+            # query the correct type name for user-defined columns
+            if data_type == 'USER-DEFINED' : 
+                sql =  sql_text(("""
+            SELECT udt_schema::text ||'.'|| udt_name::text as defined_type
+            FROM information_schema.columns
+            WHERE table_schema = '{schema}' AND column_name = '{column}' and table_name = '{table}'
+            GROUP BY defined_type
+            LIMIT 1;
+        """).format(schema = self.schema, table = self.table_name, column = attr))
```

Comment:
Whitespace/indentation looks wrong?

Reply:
It was working, but something like this may looks better ?